File: java_projects/1-Java从入门到放弃/src/Test1.java
/**
 * @author: Li Tian
 * @contact: 694317828@qq.com
 * @software: pycharm
 * @file: Test1.java
 * @time: 2019/8/20 17:32
 * @desc:
 */

public class Test1 {
    public static void main(String[] args){
        int a = 3;
        int b = 4;
        System.out.println(a&b);
        System.out.println(a|b);
        System.out.println(a^b);
        System.out.println(~a);

        // 移位
        int c = 3<<2;
        System.out.println(c);
        System.out.println(12>>1);
    }
}

File: python_projects/2- learnings自学8/test1.py
#!/usr/bin/env python
# -*- coding: UTF-8 -*-
# coding=utf-8 

"""
@author: Li Tian
@contact: litian_cup@163.com
@software: pycharm
@file: test1.py
@time: 2020/12/26 11:19
@desc: 一些简单的测试
"""

with open('test1.txt', 'a+', encoding='utf-8') as f:
    print('你好吗？', file=f)
    print('狗贼？', file=f)
